src/Autosuggest.js

```diff
@@ -243,7 +243,7 @@ class Autosuggest extends Component {
               var updatedFocusedItemIndex = newFocusedItemIndex,
                 updatedFocusedSectionIndex = newFocusedSectionIndex;
 
-              if (newFocusedSectionIndex === 0 && newFocusedItemIndex === 0) {
+              if (this.props.id === 'header-search' && newFocusedSectionIndex === 0 && newFocusedItemIndex === 0) {
                 if (event.key === 'ArrowDown') {
                   updatedFocusedItemIndex = 1;
```

Comment:
Long term Fix: Instead of forcing the second item as default, would it be better to make items as 'unselectable' and skip them in general? What if nth items was not selectable? This is the reason why we have current bug.

Reply:
Agreed. Unfortunately though, react-autosuggest doesn't support a lot of functionality that we require out-of-box (maybe somewhat obvious).

Reply:
Agreed, but we need to find correct ways of doing things and not hacks. What's your proposal for fixing this? Given its slipped few time and had to put 2 tech leads to get it on track, let's take it offline and have a through technical review on search feature as a whole. 

Please work on a design document and schedule something with all the FE tech leads.

Reply:
Ok sure. Will do.